<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1582010179922' stop='1582010227686'>
    <name>Functional TCs</name>
    <title>Functional TCs</title>
    <test-cases>
        <test-case start='1582010179924' status='failed' stop='1582010227685'>
            <name>TC_DDM_001</name>
            <title>TC_DDM_001</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (C:\Users\rso\AppData\Roaming\npm\node_modules\protractor\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at ontimeout (timers.js:436:11)
                    at tryOnTimeout (timers.js:300:5)
                    at listOnTimeout (timers.js:263:5)
                    at Timer.processTimers (timers.js:223:10)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>